docs/topics/note-preview-booster-source.adoc

```diff
@@ -1,3 +1,4 @@
 .Viewing the Booster source code and README
-To view the source code and README file of this booster, download and extract the ZIP file with the booster.
-To get the download link of the ZIP file, see the link:{link-getting-started-guide}#oso-create-booster[Creating and Deploying a Booster Using OpenShift Online] chapter of the {getting-started-guide-name}.
+To view the source code and README file of this booster, use link:{link-launcher-oso}[{launcher-oso}] or the {launcher} tool on a {OpenShiftLocal} to generate your own version of the booster. From there you can view the generated GitHub repository or download and extract the ZIP file containing the booster code.
+
+You can find more details on link:{link-getting-started-guide}#deploying-a-booster-to-oso[using {launcher-oso}] or link:{link-getting-started-guide}#osl-create-booster[the {launcher} tool on a {OpenShiftLocal}] in the link:[{getting-started-guide-name}].
```

Comment:
I've just seen this rendered and the last link in the paragraph actually doesn't lead to the Getting Started Guide, because the link is `<a href="">` (i.e., a link to the current document).

Reply:
sorry about that @Ladicek, forgot to drop the url in.

Fixing in a follow up PR

Reply:
Fixed here: https://github.com/fabric8-launcher/launcher-documentation/pull/1033

Reply:
Thanks @rhoads-zach!